<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>107</y>
      <width>240</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Перенос</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>13</y>
      <width>240</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить фигуру из файла</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dxSpinbox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>157</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dySpinbox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>157</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>137</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>по оси X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>137</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>по оси Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>137</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>по оси Z</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dzSpinbox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>157</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>53</y>
      <width>240</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сохранить модель в файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>187</y>
      <width>240</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Перенести</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kySpinbox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>272</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kzSpinbox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>272</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>226</y>
      <width>240</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Масштабирование</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>252</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>вдоль X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>252</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>вдоль Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>302</y>
      <width>240</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Масштабирование</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>252</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>вдоль Y</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kxSpinbox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>272</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>371</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Угол поворота в градусах</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="angleSpinbox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>368</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-359</number>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>341</y>
      <width>240</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>401</y>
      <width>240</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Повернуть</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="axisXRadio">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>690</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>687</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ось</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="axisYRadio">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>690</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="axisZRadio">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>690</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>630</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="cxSpinbox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>650</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="czSpinbox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>650</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="cySpinbox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>650</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>590</y>
      <width>240</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Параметры преобразований</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>630</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>630</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>442</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Проецирование</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="projectButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>538</y>
      <width>240</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Спроецировать</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>578</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="projectBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>472</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Параллельное проецирование</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Центральное проецирование</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>508</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Положение наблюдателя</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dSpinbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>928</x>
      <y>503</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>753</x>
      <y>651</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Центр</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
